docs/tutorial/aws-lambda.txt

```diff
@@ -0,0 +1,122 @@
+==============================
+Deploy to AWS Lambda with Bref
+==============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+`Bref <https://bref.sh>`__ allows to deploy serverless PHP applications on AWS Lambda.
+In this tutorial, you will deploy a simple PHP application with the MongoDB PHP extension,
+and connect to an Atlas cluster using AWS IAM authentication.
+
+Prerequisites
+--------------
+
+Before you begin, you must install Bref on your machine. You can follow the
+`official documentation to setup Bref <https://bref.sh/docs/setup>`__.
+
+Install the MongoDB extension
+-----------------------------
+
+By default, the bref layer is compiled with PHP and a few extensions. Additional extensions
+are provided in additional layers.
+
+Start by creating a new directory for your project and install the required MongoDB
+and Bref dependencies. This project will be a bare minimum PHP web application that
+connects to a MongoDB cluster.
+
+.. code-block:: bash
+
+    mkdir bref-mongodb-app && cd bref-mongodb-app
+    composer init
+    composer require bref/bref bref/extra-php-extensions mongodb/mongodb
+    vendor/bin/bref init
+
+
+The file ``index.php`` has been created. To validate the deployment, you can start
+by deploying this default application.
+
+.. code-block:: bash
+
+    serverless deploy
+
+
+Bref provides a Lambda layer with PHP and some very common extensions.
+Additional extensions are provided by the package `bref/extra-php-extension <https://github.com/brefphp/extra-php-extensions>`__.
+
+.. code-block:: yaml
+
+    plugins:
+        - ./vendor/bref/bref
+        - ./vendor/bref/extra-php-extensions
+
+    functions:
+        api:
+            handler: index.php
+            runtime: php-83-fpm
+            layers:
+                - ${bref-extra:mongodb-php-83}
+
+
+Let's try to use the MongoDB driver with this simple web page that list planets
+from the `sample dataset <https://www.mongodb.com/docs/atlas/sample-data/>`__.
+Replace the contents of ``index.php`` with the following:
+
+.. literalinclude:: /examples/aws-lambda/index.php
+   :language: php
+
+
+Deploy the application
```

Comment:
@mongodb/dbx-php What's your opinion on the formatting of this tutorial? It's actually steps.

Reply:
No objections if you feel it's worth the effort.

Please make sure that formatting doesn't clash with "AWS Credentials", which has a numbered list within. I'm not expecting any issue nesting a list within the RST step syntax -- just a request to confirm the rendered output looks presentable.

Reply:
I think skipping steps formatting is fine and more efficient with time. The issue I was focusing on was missing punctuation. Perhaps it could be something to experiment with in a future PR!

Reply:
Atlas links doesn't work. 
<img width="750" alt="image" src="https://github.com/mongodb/mongo-php-library/assets/400034/a75bc581-33df-48ad-a998-ff81bc9b4828">